lib/routes/zhihu/all-collections.ts

```diff
@@ -0,0 +1,114 @@
+import { Route, ViewType } from '@/types';
+import got from '@/utils/got';
+import { header } from './utils';
+import { parseDate } from '@/utils/parse-date';
+import cache from '@/utils/cache';
+
+export const route: Route = {
+    path: '/people/allCollections/:id',
+    categories: ['social-media'],
+    view: ViewType.Articles,
+    example: '/zhihu/people/allCollections/87-44-49-67',
+    parameters: { id: '作者 id，可在用户主页 URL 中找到' },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: true,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    radar: [
+        {
+            source: ['www.zhihu.com/people/:id'],
+            // target: 'people/allCollections/:id',
+        },
+    ],
+    name: '用户全部收藏内容',
+    maintainers: ['CohenV'],
+    handler,
+};
+
+async function handler(ctx) {
+    const id = ctx.req.param('id');
+    const apiPath = `https://api.zhihu.com/people/${id}/collections`;
+
+    const response = await got(String(apiPath), {
```

Comment:
`apiPath` is already a string. [Do not cast the same type again](https://docs.rsshub.app/joinus/advanced/script-standard#casting).